stretch_description/batch/update_description.py

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env python3
+"""
+This script updates the Mesh files and URDF file specific to a robot batch.
+"""
+import os
+from os.path import exists
+import stretch_body.hello_utils as hu
+
+if exists(hu.get_fleet_directory() + 'stretch_configuration_params.yaml'):
+    batch_name = hu.read_fleet_yaml('stretch_configuration_params.yaml')['robot']['batch_name'].lower()
+else:
+    batch_name = hu.read_fleet_yaml('stretch_re1_factory_params.yaml')['robot']['batch_name'].lower()
+
+cmd_cp_meshes = 'cp ~/catkin_ws/src/stretch_ros/stretch_description/batch/' + batch_name + '/meshes/*.STL ~/catkin_ws/src/stretch_ros/stretch_description/meshes'
+cmd_cp_urdfs = 'cp ~/catkin_ws/src/stretch_ros/stretch_description/batch/' + batch_name + '/urdf/* ~/catkin_ws/src/stretch_ros/stretch_description/meshes'
+
+print("Copying in Mesh files and URDF files from batch:" + batch_name)
+print(cmd_cp_meshes)
+os.system(cmd_cp_meshes)
+print(cmd_cp_urdfs)
+os.system(cmd_cp_urdfs)
```

Comment:
Let's improve this script by getting it to report failures instead of silently failing. Take a look at [this script](https://github.com/hello-robot/stretch_ros2/blob/bugfix/calibration_subprocess_exitcode/stretch_calibration/stretch_calibration/update_with_most_recent_calibration.py#L9-L12) from ROS2 for how to use the `subprocess` module to execute a bash command and check the return code to exit with `sys.exit(1)` when the command fails to execute.

In this case, if the cmds fail, it means the batch name was not valid. An error should relay this to the user.

Beyond improving the user experience, this would enable the Stretch Install scripts to halt installation if the `update_description.py` script fails to execute successfully.

Reply:
Thanks for your input, and I agree using `subprocess` should be our go-to method here. I have pushed the requested changes. This branch should now be good to be merged.